src/GenerativeVegetable.java

```diff
@@ -0,0 +1,19 @@
+public class GenerativeVegetable extends Vegetable {
```

Comment:
There is a principle in programming called DRY - do not repeat yourself. In your example, to print out vegetable info, you have to reprint super class(Vegetable) properties for every child class. But what if we add new property to parent class? We will be forced to modify each child class to print out new information. Think how to make it more OOP.